src/input/plugins/CdioParanoiaMusicBrainzXMLParser.hxx

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "CdioParanoiaMusicBrainzTags.hxx"
+
+class MusicBrainzXMLParser
```

Comment:
What, a class that has no fields and only one method - that design doesn't appear to make any sense.